setup.py

```diff
@@ -98,7 +100,21 @@ def run(self):
 
 def read_requirements(path):
     with open(os.path.join(BASE_DIR, path)) as f:
-        _pipenv_flags, *requirements = f.read().split('\n')
+        _pipenv_flags, *lines = f.read().split('\n')
+
+    # Transforms VCS requirements to PEP 508
```

Comment:
:exploding_head: - A real testament to the current state of python dependency management.

Reply:
I must admit this patching is too hacky to love. Ideally this code can be removed once there is a solution to the dependency issue.  Let's include a TODO at minimum.

Reply:
✔️